internal/backends/postgresql/postgresql.go

```diff
@@ -39,7 +39,13 @@ type stats struct {
 }
 
 // collectionsStats returns statistics about tables and indexes for the given collections.
+//
+// If the list of collections is empty, then stats filled with zero values is returned.
```

Comment:
This fixes a situation when the empty list is provided, and the `SELECT` query below returns an invalid syntax error.  There were some tests that were failing because of it.